composer.json

```diff
@@ -39,15 +39,16 @@
         }
     ],
     "require": {
-        "php": ">=7.1.3",
-        "illuminate/console": "^5.8",
-        "illuminate/database": "^5.8",
-        "illuminate/filesystem": "^5.8"
+        "php": ">=7.2",
+        "illuminate/console": "^6.0",
+        "illuminate/database": "^6.0",
+        "illuminate/filesystem": "^6.0",
+        "laravel/helpers": "^1.1"
```

Comment:
Hi @divdax, is this not intended to add?
I merge PR with this to remove https://github.com/owen-it/laravel-auditing/pull/557